modules/ossm-deploy-cluster-wide-control-plane-console.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-create-smcp.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-deploy-cluster-wide-control-plane-console_{context}"]
+= Configuring the control plane for cluster-wide deployment with the web console
+
+You can configure the `ServiceMeshControlPlane` resource for cluster-wide deployment using the {product-title} web console.  In this example, `istio-system` is the name of the {SMProductShortName} control plane project.
+
+.Prerequisites
+
+* The {SMProductName} Operator must be installed.
+* An account with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} web console as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
+
+. Create a project named `istio-system`.
++
+.. Navigate to *Home* -> *Projects*.
++
+.. Click *Create Project*.
++
+.. In the *Name* field, enter `istio-system`. The `ServiceMeshControlPlane` resource must be installed in a project that is separate from your microservices and Operators.
```

Comment:
Maybe we can say here:
In the *Name* field, enter `istio-system`. *Important*: the `ServiceMeshControlPlane` resource must be installed in a project that is separate from your microservices and Operators.
This is only a suggestion, I think will be more clear for the customers

Reply:
We discussed. NA due to tooling.